blueocean-rest-impl/src/main/java/io/jenkins/blueocean/service/embedded/TryBlueOceanMenu.java

```diff
@@ -62,7 +64,7 @@
     }
 
     private boolean exists(Actionable actionable, BlueOceanUrlAction blueOceanUrlAction){
-        for(Action a: actionable.getActions()) {
+        for(Action a: actionable.getAllActions()) {
```

Comment:
@imeredith We can't call `actionable.getAllActions()` because it calls `actionable.createFor()` so there is loop. Due to this many tests are failing resulting in to missing coverage.

```
Tests run: 8, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 1.257 sec - in io.jenkins.blueocean.util.HttpRequestTest

Results :

Failed tests: 
  PipelineApiTest.parameterizedFreestyleTest:760->BaseTest.get:102->BaseTest.get:107->BaseTest.get:124 expected:<200> but was:<404>
Tests in error: 
  ArtifactContainerImplTest.testArtifactsListing:23->Object.wait:502->Object.wait:-2 » TestTimedOut
  BlueOceanWebURLBuilderTest.test_freestyle:84->Object.wait:502->Object.wait:-2 » TestTimedOut
  LinkResolverTest.runLinkResolveTest:54->Object.wait:502->Object.wait:-2 » TestTimedOut
  PipelineApiTest.actionsTest:803->Object.wait:502->Object.wait:-2 » TestTimedOut
  PipelineApiTest.findAllPipelineTest:422->Object.wait:502->Object.wait:-2 » TestTimedOut
  PipelineApiTest.findPipelineRunsForAPipelineTest:395->Object.wait:502->Object.wait:-2 » TestTimedOut
  PipelineApiTest.findPipelineRunsForAllPipelineTest » Reactor java.lang.Error: ...
  PipelineApiTest.getFreeStyleJobTest:251->Object.wait:502->Object.wait:-2 » TestTimedOut
  PipelineApiTest.getPipelineRunLatestTest » Runtime Unexpected issues encounter...
  PipelineApiTest.getPipelineRunTest:313->Object.wait:502->Object.wait:-2 » TestTimedOut
  PipelineApiTest.getPipelineRunWithTestResult:294->Object.wait:502->Object.wait:-2 » TestTimedOut
  PipelineApiTest.getPipelineRunsStopTest:374->Object.wait:502->Object.wait:-2 » TestTimedOut
  PipelineApiTest.getPipelineRunsTest:344->Object.wait:502->Object.wait:-2 » TestTimedOut
  PipelineApiTest.getPipelinesTest:186->Object.wait:502->Object.wait:-2 » TestTimedOut
  PipelineApiTest.shouldFailToGetRunForInvalidRunId1:358->Object.wait:502->Object.wait:-2 » TestTimedOut
  PipelineApiTest.testArtifactsRunApi » Reactor java.lang.AssertionError: This m...
  PipelineApiTest.testNewPipelineQueueItem:556->Object.wait:502->Object.wait:-2 » TestTimedOut
  PipelineApiTest.testPipelineQueue:526->Object.wait:502->Object.wait:-2 » TestTimedOut
  TryBlueOceanMenuTest.testOpenBlueOcean:20 » ElementNotFound elementName=[a] at...
  BlueJUnitTestResultTest.createsTestResult:34->Object.wait:502->Object.wait:-2 » TestTimedOut
  BlueTrendsApiTest.getJUnitTrends:49->Object.wait:502->Object.wait:-2 » TestTimedOut

Tests run: 123, Failures: 1, Errors: 21, Skipped: 7
```

Reply:
Oh i see, thanks